universal-login-relayer/lib/config/config.prod.ts

```diff
@@ -18,7 +18,7 @@ export const config: Config =  Object.freeze({
   ],
   walletContractAddress: getEnv('WALLET_MASTER_ADDRESS', ''),
   contractWhiteList: {
-    wallet: [],
+    wallet: ['0x681db2a2d0448e46edf80b271677300548eff7255414d34c02a34839bd7bf9b7'],
```

Comment:
This is copied from:
https://github.com/UniversalLogin/UniversalLoginSDK/blob/40899e5d6a3b559ee961ae224c2e9400bfa8fb4e/universal-login-relayer/lib/http/relayers/RelayerUnderTest.ts#L87

`WalletMasterWithRefund ` is:
https://github.com/UniversalLogin/UniversalLoginSDK/blob/40899e5d6a3b559ee961ae224c2e9400bfa8fb4e/universal-login-relayer/lib/http/relayers/RelayerUnderTest.ts#L6

`Wallet.json` is used after rebuilding everything using `yarn ci`

Reply:
👍 
btw:
- could be address instead of hash. I guess hash is a bit better probably cause it reusable across environments.
- whenever you see `WalletMasterWithRefund` you can rename to `Wallet` or `WalletContract`.